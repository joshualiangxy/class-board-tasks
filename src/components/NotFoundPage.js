import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
  <div>
    <h2>Nothing here to look at</h2>
    <Link to="/" />
  </div>
);

export default NotFoundPage;
