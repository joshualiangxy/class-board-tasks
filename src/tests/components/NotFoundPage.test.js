import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

const wrapper = shallow(<NotFoundPage />);

describe('render', () => {
  it('should render NotFoundPage', () => expect(wrapper).toMatchSnapshot());
});

describe('home link', () => {
  it('should link to home page', () =>
    expect(wrapper.find('Link').prop('to')).toBe('/'));
});
