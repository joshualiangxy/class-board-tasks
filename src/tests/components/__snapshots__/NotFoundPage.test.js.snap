// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render NotFoundPage 1`] = `
<div>
  <h2>
    Nothing here to look at
  </h2>
  <Link
    to="/"
  >
    Home
  </Link>
</div>
`;
